finagle-core/src/main/scala/com/twitter/finagle/util/LoadService.scala

```diff
@@ -174,6 +177,9 @@ private object ClassPath {
   }
 }
 
+object ignoredPaths extends GlobalFlag(Seq.empty[String],
```

Comment:
do you mind making this flag's name more specific? 

right now its "com.twitter.finagle.util.ignoredPaths" which is a bit opaque for my tastes. perhaps "com.twitter.finagle.util.loadServiceIgnoredPaths" ?